### PR TITLE
Add Viewport Meta Tag to Get Bootstrap Working on Mobile Devices

### PR DESCRIPTION
**Pull request by iarobinson**
### Description of changes:

This just adds the meta tag for mobile devices to work with Bootstrap

### This PR addresses issue #3688 

In your PR did you:

### From this:
Screen Shot 2020-09-30 at 5 24 09 PM

### To this:
Screen Shot 2020-09-30 at 5 24 48 PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -54,4 +54,10 @@ bower.json
...
 # Ignore database.yml from version control...
-config/database.yml
\ No newline at end of file
+config/database.yml
+
+config/initializers/s3.rb
+config/sitemap.rb
+
+config/initializers/recaptcha.rb
+db/structure.sql
```
Comment (rlgreen91) [inline .gitignore:63]: We don't want to ignore structure.sql, as we are currently using this in place of a schema.rb file.

**Review by rlgreen91 — CHANGES_REQUESTED** [review]
Looks good, the big thing is that we don't want to add `db/structure.sql` to the .gitignore file.

File: .gitignore (hunk 1 above)
Comment (iarobinson) [reply .gitignore]: Got it!

**Comment by iarobinson** [comment]
Here is the error that Travis is reporting for why the tests are failing:

```
before_script.1
0.13s$ psql -c 'create database travis_ci_test;' -U postgres
before_script.2
0.01s$ cp config/database.ci.yml config/database.yml
5.54s$ bundle exec rake db:create db:migrate db:structure:load
Database 'travis_ci_test' already exists
DEPRECATION WARNING: ActiveRecord::Base.raise_in_transactional_callbacks= is deprecated, has no effect and will be removed without replacement. (called from <top (required)> at /home/travis/build/EBWiki/EBWiki/config/environment.rb:8)
rake aborted!
ArgumentError: Missing required arguments: aws_access_key_id, aws_secret_access_key
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/fog-core-1.45.0/lib/fog/core/service.rb:244:in `validate_options'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/fog-core-1.45.0/lib/fog/core/service.rb:268:in `handle_settings'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/fog-core-1.45.0/lib/fog/core/service.rb:98:in `new'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/fog-core-1.45.0/lib/fog/core/services_mixin.rb:16:in `new'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/fog-core-1.45.0/lib/fog/storage.rb:27:in `new'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/carrierwave-1.3.1/lib/carrierwave/uploader/configuration.rb:126:in `eager_load_fog'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/carrierwave-1.3.1/lib/carrierwave/uploader/configuration.rb:139:in `fog_credentials='
/home/travis/build/EBWiki/EBWiki/config/initializers/s3.rb:4:in `block in <top (required)>'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/carrierwave-1.3.1/lib/carrierwave/uploader/configuration.rb:161:in `configure'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/carrierwave-1.3.1/lib/carrierwave.rb:14:in `configure'
/home/travis/build/EBWiki/EBWiki/config/initializers/s3.rb:3:in `<top (required)>'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:287:in `load'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:287:in `block in load'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:259:in `load_dependency'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:287:in `load'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/engine.rb:648:in `block in load_config_initializer'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/notifications.rb:166:in `instrument'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/engine.rb:647:in `load_config_initializer'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/engine.rb:612:in `block (2 levels) in <class:Engine>'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/engine.rb:611:in `each'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/engine.rb:611:in `block in <class:Engine>'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/initializable.rb:30:in `instance_exec'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/initializable.rb:30:in `run'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/initializable.rb:55:in `block in run_initializers'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/initializable.rb:44:in `each'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/initializable.rb:44:in `tsort_each_child'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/initializable.rb:54:in `run_initializers'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/application.rb:352:in `initialize!'
/home/travis/build/EBWiki/EBWiki/config/environment.rb:8:in `<top (required)>'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:293:in `require'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:293:in `block in require'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:259:in `load_dependency'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/activesupport-5.0.7.2/lib/active_support/dependencies.rb:293:in `require'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/application.rb:328:in `require_environment!'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/railties-5.0.7.2/lib/rails/application.rb:448:in `block in run_tasks_blocks'
/home/travis/build/EBWiki/EBWiki/vendor/bundle/ruby/2.6.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
/home/travis/.rvm/gems/ruby-2.6.5/bin/ruby_executable_hooks:24:in `eval'
/home/travis/.rvm/gems/ruby-2.6.5/bin/ruby_executable_hooks:24:in `<main>'
Tasks: TOP => db:migrate => environment
(See full trace by running task with --trace)
The command "bundle exec rake db:create db:migrate db:structure:load" failed and exited with 1 during .
Your build has been stopped.
```

Anyone have any idea what is going on here?

**Comment by rlgreen91** [comment]
@iarobinson I'm not too concerned about these failing tests - Travis doesn't inject environment variables into builds properly when the PR comes from a fork, for security reasons.  You should receive an invite to join as a collaborator soon - that will allow you to make branches from the repo itself without creating a fork.

Given that, and the nature of this fix, I'm going to go ahead and review it as is.

**Review by rlgreen91 — APPROVED** [review]

**Pull request merged**